modules/creating-the-training-job.adoc

```diff
@@ -0,0 +1,183 @@
+
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+ifndef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-on-data-science-projects_nb-server#creating-a-data-science-project_nb-server[Creating a data science project].
+endif::[]
+ifdef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#_using_data_science_projects[Creating a data science project].
+endif::[]
+
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+* You have access to a model.
+* You have access to data to train the model.
+
+.Procedure
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<admin_username>__ -p __<password>__
+----
+
+. Define a `ConfigMap` object in a YAML file called `config_trainingjob.yaml` with the following contents:
++
+[source]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: my-config-trainingjob
+  namespace: kfto
+data:
+  config.json: |
+    {
+      "model_name_or_path": "bigscience/bloom-560m",
+      "training_data_path": "/data/input/twitter_complaints.json",
+      "output_dir": "/data/output/tuning/bloom-twitter",
+      "num_train_epochs": 10.0,
+      "per_device_train_batch_size": 4,
+      "gradient_accumulation_steps": 4,
+      "learning_rate": 1e-05,
+      "response_template": "\n### Label:",
+      "dataset_text_field": "output",
+      "use_flash_attn": false
+    }
+
+----
++
+The `my-config-trainingjob` object configures the parameters of the training job, as follows:
+
+* The `model_name_or_path` field specifies the name of the pre-trained model.
```

Comment:
Suggestion: to simplify text - could change this to a table 
Field | Description
`model_name_or_path` |  Specifies the name of the pre-trained model
.
.
.

Might make it easier for reader to scan?

Reply:
I think I misunderstood an earlier comment where I thought you were suggesting moving these items to substeps, which is what I did. Do you think it would be better to use a table instead?